Sasha/Files/pdf_generator/test_pdf_gen.py

```diff
@@ -0,0 +1,20 @@
+from pdf_gen import open_text_file
+from fpdf import FPDF
+import unittest
+
+class TestPDFGen(unittest.TestCase):
```

Comment:
Add test for more bigger amount pf text to convert
1) When second page will be used
2) When there is a big single line